sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/internal/scan/ClientApiFacade.java

```diff
@@ -234,6 +235,18 @@ public ApiResponse importOpenApiFile(String openApiFile, String url, String cont
         return clientApi.openapi.importFile(openApiFile, url, contextId);
     }
 
+    /**
+     *
```

Comment:
Please: Either remove the javadoc part or write a text and describe the parameters.